wandb/deploys/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-bullseye
+
+RUN apt-get update && apt-get upgrade && apt-get install
+# install git
+RUN apt-get update && apt-get upgrade -y \
+    && apt-get install -y git \
+    && apt-get -qy autoremove \
+    && apt-get clean && rm -r /var/lib/apt/lists/*
+
+# required pip packages
+RUN pip install --no-cache-dir git+https://github.com/wandb/client.git@launch/general-launch-agent wandb[launch] kubernetes
```

Comment:
This should be: `git+https://github.com/wandb/client.git@launch/general-launch-agent#egg=wandb[launch]`

Reply:
I don't think so, we want to install this branch of wandb client and the wandb launch option